sktm/patchwork.py

```diff
@@ -959,7 +963,7 @@ def __get_mbox_url_sfx(self):
         Returns:
             The patch mbox URL suffix.
         """
-        if self.fields:
+        if self.is_rh_fork:
```

Comment:
I wonder, should we move this to the `PatchworkProject` class, and use it in `get_patch_message()`?

Reply:
That makes sense, and would simplify the code around `get_patch_message` calls. However, I'd definitely do it as a separate pull.

Reply:
OK, would be great if you could do this later, then. Thank you :)